web/docs/tql2/operators/load_google_cloud_pubsub.md

```diff
@@ -13,7 +13,8 @@ The connector tries to retrieve the appropriate credentials using Google's
 
 ## Description
 
-The `google_cloud_pubsub` loader acquires raw bytes from a Google Cloud Pub/Sub subscription.
+The `google_cloud_pubsub` loader acquires raw bytes from a Google Cloud Pub/Sub
+subscription.
```

Comment:
I think we shouldn't be using the "format name" in backticks anymore, as there is no entity `google_cloud_pubsub`. We can probably just remove the name here, it's clear where we are.